docs/integrations/sources/postgres.md

```diff
@@ -97,5 +97,82 @@ GRANT SELECT ON ALL TABLES IN SCHEMA <schema_name> TO airbyte;
+* If your data set is small and you just want snapshot of your table in the destination, consider using Full Refresh replication for your table instead of CDC.
+* If the limitations prevent you from using CDC and your goal is to maintain a snapshot of your table in the destination, consider using non-CDC incremental and occasionally reset the data and re-sync.
+
+### CDC Limitations
+* Make sure to read our [CDC docs](../../architecture/cdc.md) to see limitations that impact all databases using CDC replication.
+* CDC is only available for Postgres 10+.
+* Airbyte requires a replication slot configured only for its use. Only one source should be configured that uses this replication slot.
+* Log-based replication only works for master instances of Postgres.
+* Using logical replication increases disk space used on the database server. The additional data is stored until it is consumed.
+
+### Setting up CDC for Postgres
+
+Follow one of these guides to enable logical replication:
+* [Bare Metal, VMs (EC2/GCE/etc), Docker, etc.](#setting-up-cdc-on-bare-metal-vms-ec2gceetc-docker-etc)
+* [AWS Postgres RDS or Aurora](#setting-up-cdc-on-aws-postgres-rds-or-aurora)
+* [Azure Database for Postgres](#setting-up-cdc-on-azure-database-for-postgres)
+
+Then, the Airbyte user for your instance needs to be granted `REPLICATION` and `LOGIN` permissions. Since we are using embedded Debezium under the hood for Postgres, we recommend reading the [permissioning section of the Debezium docs](https://debezium.io/documentation/reference/connectors/postgresql.html#postgresql-permissions) for more information on what is required.
+
+Finally, you will need to create a replication slot. Here is the query used to create a replication slot called `airbyte_slot`: 
+```
+SELECT pg_create_logical_replication_slot('airbyte_slot', 'pgoutput');`
+```
+
+This slot **must** use `pgoutput`.
+
+After providing the name of this slot when configuring the source, you should be ready to sync data with CDC!
+
+### Setting up CDC on Bare Metal, VMs (EC2/GCE/etc), Docker, etc.
+Three settings must be configured in the `postgresql.conf` file for your database:
+```
+wal_level = logical             
+max_wal_senders = 1             
+max_replication_slots = 1
+```
+
+* `wal_level` is the type of coding used within the Postgres write-ahead log. This must be set to `logical` for Airbyte CDC.
+* `max_wal_senders` is the maximum number of processes used for handling WAL changes. This must be at least one.
+* `max_replication_slots` is the maximum number of replication slots that are allowed to stream WAL changes. This must one if Airbyte will be the only service reading subscribing to WAL changes or more if other services are also reading from the WAL.
+
+After setting these values you will need to restart your instance.
+
+Finally, [follow the rest of steps above](#setting-up-cdc-for-postgres).
+
+### Setting up CDC on AWS Postgres RDS or Aurora
```

Comment:
is GCP cloudSQL compatible with our solution? i'm assuming it's not since you didn't add it. if so i think we should say it?

Reply:
Added a section and a link to https://issuetracker.google.com/issues/120274585